From: Yifeng <wuyifeng@nd.com.cn>
Date: Sat, 2 Jul 2022 07:40:00 +0000
Subject: AP9-5820 Chromium Menu in French is not fully translated and some text is cropped

Fix version: 1.1.0
Hide the 'View source' from the menu.
Hide the 'Exit' from the menu.

---
 chrome/android/java/res/menu/main_menu.xml | 2 --
 chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java | 2 ++
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/res/menu/main_menu.xml b/chrome/android/java/res/menu/main_menu.xml
--- a/chrome/android/java/res/menu/main_menu.xml
+++ b/chrome/android/java/res/menu/main_menu.xml
@@ -144,8 +144,6 @@
         <item android:id="@+id/get_image_descriptions_id"
             android:title="@string/menu_get_image_descriptions"
             android:icon="@drawable/ic_image_descriptions"/>
-        <item android:id="@id/divider_line_id"
-            android:title="@null" />
         <item android:id="@+id/reader_mode_prefs_id"
             android:title="@string/menu_reader_mode_prefs"
             android:icon="@drawable/reader_mode_prefs_icon" />
diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
@@ -522,6 +522,8 @@ public class AppMenuPropertiesDelegateImpl implements AppMenuPropertiesDelegate
         menu.findItem(R.id.all_bookmarks_menu_id).setVisible(false);
         menu.findItem(R.id.bookmark_all_tabs_menu_id).setVisible(false);
         menu.findItem(R.id.preferences_id).setVisible(false);
+        menu.findItem(R.id.view_source_id).setVisible(false);
+        menu.findItem(R.id.exit_id).setVisible(false);

         updateManagedByMenuItem(menu, currentTab);
     }
--
2.17.1